### DISCUSSION TITLE
macOS: "Do you want to terminate running processes in this window" dialog window

### DISCUSSION THREAD

**Discussion by johnmapeson** [fish-shell/fish-shell#10886] [Q&A] [answered]
When I try to close a macOS terminal window with fish running, the following dialog message appears:

<img width="241" alt="image" src="https://github.com/user-attachments/assets/cffcd631-8d57-48e7-a148-cae04c2230db">

Is it possible to turn it off? That is, when I press Command-W or Command-Q, the window will be closed without asking me any questions.

**Comment by zanchey** [comment, marked as answer]
In Settings > Profiles > Shell, add "fish" to the "Ask before closing" list.
